Graphical abstract and sequence features figure (#21)
greenelab/biopriors-review: content/05.network-models.md

Rather than operating on sequences, many machine learning models in biomedicine operate on inputs that lack intrinsic order.
Models may make use of gene expression data matrices from RNA sequencing or microarray experiments in which rows represent samples and columns represent genes.
To account for relationships between genes, one might incorporate known interactions or correlations when making predictions or generating a low-dimensional representation of the data (Figure {@fig:network_models}).
This is comparable to the manner in which sequence context pushes models to consider nearby base pairs similarly.

> ![When applying machine learning to gene expression data, it can be helpful to incorporate structure describing the relationships between genes. One common approach is to use a network or collection of gene sets to embed the data in a lower-dimensional space, in which genes that are in the same gene sets or that are well-connected in the network have a similar representation in the lower-dimensional space. The embedded data can then be used for classification or clustering tasks.](images/network_models.svg){#fig:network_models .white}

Reviewer: ```suggestion
![The relationships between genes provide structure that can be incorporated into machine learning models. One common approach is to use a network or collection of gene sets to embed the data in a lower-dimensional space, in which genes that are in the same gene sets or that are well-connected in the network have a similar representation in the lower-dimensional space. The embedded data can then be used for classification or clustering tasks.](images/network_models.svg){#fig:network_models .white}
```
